### ISSUE TITLE
Assistance in debugging

### ISSUE THREAD

**Issue by tomerpetel** [KeepSafe/ReLinker#105]
Hi,

Many thanks for this helpful library. My minSdkVersion is 23 so theoretically I don't need to use ReLinker but I've found it somehow during my debugging. My issue is even weirder as I see crashes in low percentage (~30 out of 1M users) and when digging deeper I can see that it can crash but afterwards the user manages to register successfully and use the application. I tried to implement a loadLib() method which recessively tries to load the lib.

So long story short, the SO file must be there but for some reason sometimes it's not getting loaded. Any chance it's due to the fact that I'm using AAB? 

Any new direction will be greatly appreciated since I'm out of ideas :)
